packages/ios_platform_images/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0
```

Comment:
What part of this change is client-breaking?

Reply:
No change will break for the client.

Reply:
Okay, this should be 0.2.3 as a new version then (per the second-to-last paragraph [here](https://dart.dev/tools/pub/versioning#semantic-versions)).